### ISSUE TITLE
How to get hyperparameters and data augmentation methods

### ISSUE THREAD

**Issue by Zzzzzzzzzzh** [MIC-DKFZ/nnUNet#472]
Hi! This is a great paper! And I have a question to ask you.

I have run nnUNet on my data set, but I want to see some hyperparameters after running the results, such as patchsize, target spacing and the specific method of data augmentation methods (or nnUNet has the same data enhancement method for all data sets?). How should I get these results?

Thank you!

**Comment by FabianIsensee** [comment]
HI, you can see these parameters when you start a training. It will print something like this:

> stage:  0
> {'batch_size': 2, 'num_pool_per_axis': [5, 5, 5], 'patch_size': array([128, 128, 128]), 'median_patient_size_in_voxels': array([195, 207, 207]), 'current_spacing': array([2.473119  , 1.89831205, 1.89831205]), 'original_spacing': array([1.        , 0.76757812, 0.76757812]), 'do_dummy_2D_data_aug': False, 'pool_op_kernel_sizes': [[2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2]], 'conv_kernel_sizes': [[3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3]]}
> 
> stage:  1
> {'batch_size': 2, 'num_pool_per_axis': [5, 5, 5], 'patch_size': array([128, 128, 128]), 'median_patient_size_in_voxels': array([482, 512, 512]), 'current_spacing': array([1.        , 0.76757812, 0.76757812]), 'original_spacing': array([1.        , 0.76757812, 0.76757812]), 'do_dummy_2D_data_aug': False, 'pool_op_kernel_sizes': [[2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2]], 'conv_kernel_sizes': [[3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3]]}
> 
> I am using stage 1 from these plans
> I am using batch dice + CE loss
> 


The data augmentation is always the same (but it's different for 2d and 3d data!). There is just one thing that is adapted: for anisotropic datasets the spatial augmentations (scaling, rotation) are done only in-plane
